packages/theming-core/README.md

```diff
@@ -0,0 +1,17 @@
+# [@uifabric/theming-core](http://dev.microsoft.com/fabric)
+
+##### ThemingCore components for Office UI Fabric React
+
+These are not production ready components and __should never be used in product__. This ThemingCore space is useful for testing new components whose APIs might change before final release.
```

Comment:
It might be good to define what the theming core package intent is, and what the main api surface is.